mp4parse_capi/src/lib.rs

```diff
@@ -334,12 +334,17 @@ pub unsafe extern fn mp4parse_get_track_info(parser: *mut mp4parse_parser, track
                                      context.timescale,
                                      track.duration) {
         info.media_time = track.media_time.map_or(0, |media_time| {
-            track_time_to_ms(media_time, track_timescale) as i64
+            track_time_to_us(media_time, track_timescale)
+                .unwrap_or(0) as i64
```

Comment:
You can't use unwrap_or here, this failure needs to turn into an MP4PARSE_ERROR return.